test/testinput/bufr_ncep_snow_adpsfc.yaml

```diff
@@ -49,56 +49,56 @@ observations:
       obsdataout: "./testrun/gdas.t06z.adpsfc_snow.tm00.nc"
 
       variables:
-        - name: "dateTime@MetaData"
+        - name: "MetaData/dateTim"
```

Comment:
I think you have a typo - replace dateTim with dateTim**e**

Reply:
Thank you for your checking. Yes, it is a typo.  Now I fixed it.